### PR TITLE
fix: reduce the number of accessibility issues

### PR DESCRIPTION
**Pull request by csouchet**
With the help of the POC #852, we have been able to detect some issues on the accessibility of the PA website.

Here are the fixes.

### CODE REVIEW

File: src/theme/components/Layout.tsx [hunk 1]
```diff
@@ -61,20 +61,20 @@ const Layout = ({ children }: Props): JSX.Element => (
...
           }}
           contentStyle={{
-            color: theme.colors.background,
+            color: '#FFFFFF',
           }}
           cookieValue={false}
           buttonStyle={{
             background: '#90EE90',
-            color: theme.colors.primary,
+            color: '#000000',
```
Comment (tbouffard) [inline src/theme/components/Layout.tsx:72]: ❓ if we remove the usage of the theme colors, what is the point of defining them? are they used elsewhere?

**Review by tbouffard — COMMENTED** [review]
Some fixes are unclear to me. Is it possible to describe them shortly? I mean these involving color and opacity changes.
Are there any visual changes implied by the fixes provided here?

File: src/theme/components/Layout.tsx (hunk 1 above)
Comment (csouchet) [reply src/theme/components/Layout.tsx:72]: For the consent bar, we use specific colors, not used elsewhere.  
I don't see the interest to use the theme for this. 

  
:information\_source: More over, I will change the component of the Consent bar and its theme soon. So, I didn't want to spend more time on this topic :slightly\_smiling\_face:

File: src/components/seo.tsx [hunk 2]
```diff
@@ -59,6 +59,7 @@ export const SEO = ({
 
   return (
     <ThemeProvider theme={theme}>
+      <html lang="en" />
```
Comment (csouchet) [inline src/components/seo.tsx:62]: To fix the language accessibility error.

File: src/theme/components/Footer.tsx [hunk 3]
```diff
@@ -81,7 +81,14 @@ export const Footer = ({ content }: FooterProps): JSX.Element => {
                       mb={2}
                     />
                   </GatsbyLink>
-                  <MKTypography variant="h5">{brand.name}</MKTypography>
+                  <MKTypography
```
Comment (csouchet) [inline src/theme/components/Footer.tsx:84]: To fix the accessibility error for the title hierarchy.

File: src/theme/components/Layout.tsx [hunk 4]
```diff
@@ -61,20 +61,20 @@ const Layout = ({ children }: Props): JSX.Element => (
           enableDeclineButton={true}
```
Comment (csouchet) [inline src/theme/components/Layout.tsx:61]: To fix the accessibility error of the contrast of compatible colors  : https://dequeuniversity.com/rules/axe/4.6/color-contrast-enhanced

File: src/theme/components/Section.tsx [hunk 5]
```diff
@@ -43,15 +43,15 @@ const Container = ({
...
+  name: SECTION;
   icon?: string;
   label?: string;
 };
 
 const Header = ({ name, icon, label }: HeaderProps): JSX.Element => (
   <Slide direction="left" triggerOnce>
     <Heading color="text" mb={4}>
-      <Link selected>
+      <Link selected href={`#${getSectionHref(name)}`}>
```
Comment (csouchet) [inline src/theme/components/Section.tsx:54]: To fix the accessibility error for link without `href`.

File: src/theme/components/mailingListSubscription/Icon.tsx [hunk 6]
```diff
@@ -31,6 +31,11 @@ export const Icon = (props: IconProps): JSX.Element => {
   return (
     <Button
       as={props.withWiggleAnimation ? StyledOpenIcon : StyledIcon}
+      title={
```
Comment (csouchet) [inline src/theme/components/mailingListSubscription/Icon.tsx:34]: To fix the accessibility error for link without `title`.

File: src/theme/components/Layout.tsx (hunk 1 above)
Comment (tbouffard) [reply src/theme/components/Layout.tsx:72]: If we are changing the whole Consent bar, this is probably extra work to change the color here.
Anyway, I'll check the rendering to see the visual change which is what matters most here.

**Review by tbouffard — COMMENTED** [review]
Consent bar

production | commit 2c0a845
------ | ------
image | image

**Review by tbouffard — APPROVED** [review]

**Pull request merged**